spec/test_spec.rb

```diff
@@ -0,0 +1,107 @@
+# rubocop:disable Layout/LineLength
+# spec/test_spec.rb
+require './bin/main'
+require './lib/game_logic'
+require 'rainbow'
```

Comment:
When I make this it returns that error in terminal: Failure/Error: require_relative './lib/game_logic'
And I found in The Odin project tutorial that they use require not require_relative :
(#spec/calculator_spec.rb
require './lib/calculator'  #=> add this) 
https://www.theodinproject.com/courses/ruby-programming/lessons/introduction-to-rspec
so how can I use it? Thanks in advance

Reply:
@AhmedAmin90 that's because you need to use 2 dots at the start, require relative will scan from the folder the file is in, so it `./lib/calculator` means `spec/lib/calculator` but `../lib/calculator` would work fine!
![image](https://user-images.githubusercontent.com/11758151/103106905-c3621f80-4639-11eb-89b8-87aa1f47ee9d.png)

Reply:
Yes!, now it works.
Thanks a lot for this clarification.